composer.json

```diff
@@ -16,7 +16,7 @@
     "require": {
         "php": ">=7.2",
         "ext-dom": "*",
-        "guzzlehttp/guzzle": "^6.0.0",
+        "guzzlehttp/guzzle": "^6.0.0|^7.0.0",
```

Comment:
I am not sure what the behaviour of the single pipe is (seems to work), but please change it to a double pipe to be consistent with the other required packages in this file and with the composer docs: https://getcomposer.org/doc/articles/versions.md#version-range.

Reply:
Sure, no problem @mvdbos. One second.

Reply:
Done @mvdbos